lib/src/element_type.dart

```diff
@@ -71,12 +71,17 @@ abstract class ElementType extends Privacy {
 
   String get linkedName;
 
-  String get name => type.name ?? type.element.name;
+  String get name => type.element.name;
```

Comment:
FYI, `element` might be `null`, e.g. for `FunctionType`.

Reply:
Moved implementation to where an element is guaranteed.